src/api.js

```diff
@@ -660,32 +661,48 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
     * by a semicolon)
     *
     * ## Example use
-    * We have the following table, named *test* :
+    * We will create the following table, named *test* and query it with a
+    * multi-line statement using params:
     *
     * | id | age |  name  |
     * |:--:|:---:|:------:|
     * | 1  |  1  | Ling   |
     * | 2  |  18 | Paul   |
-    * | 3  |  3  | Markus |
     *
     * We query it like that:
     * ```javascript
     * var db = new SQL.Database();
-    * var res = db.exec("SELECT id FROM test; SELECT age,name FROM test;");
+    * var res = db.exec(
+    *     "DROP TABLE IF EXISTS test;\n"
+    *     + "CREATE TABLE test (id INTEGER, age INTEGER, name TEXT);"
+    *     + "INSERT INTO test VALUES ($id1, :age1, @name1);"
+    *     + "INSERT INTO test VALUES ($id2, :age2, @name2);"
+    *     + "SELECT id FROM test;"
+    *     + "SELECT age,name FROM test WHERE id=$id1",
+    *     {
+    *         "$id1": 1, ":age1": 1, "@name1": "Ling",
+    *         "$id2": 2, ":age2": 18, "@name2": "Paul"
+    *     }
+    * );
     * ```
     *
     * `res` is now :
     * ```javascript
     *     [
-    *         {columns: ['id'], values:[[1],[2],[3]]},
-    *         {columns: ['age','name'], values:[[1,'Ling'],[18,'Paul'],[3,'Markus']]}
+    *         {"columns":["id"],"values":[[1],[2]]},
+    *         {"columns":["age","name"],"values":[[1,"Ling"]]}
     *     ]
     * ```
     *
-    * @param {string} sql a string containing some SQL text to execute
+    @param {string} sql a string containing some SQL text to execute
+    @param {any[]} [params=[]] When the SQL statement contains placeholders,
```

Comment:
```suggestion
    @param {Statement.BindParams} [params=[]] When the SQL statement contains placeholders,
```

Reply:
applied suggestion in commit f2d5ad2c1c5e0c3093c69811bf8ddc772413736c